config/yaml_config.go

```diff
@@ -45,6 +45,7 @@ func (a *AgentConfig) loadNetworkYamlConfig(path string) error {
 	}
 
 	a.NetworkBPFDebug = config.Datadog.GetBool(key(netNS, "bpf_debug"))
+	a.ExcludedBPFLinuxVersions = config.Datadog.GetStringSlice(key(netNS, "excluded_linux_versions"))
```

Comment:
You'll want to wrap this with a `isSet` check

Reply:
Won't it just default to false ?

Reply:
It'll also throw an trace error since there's no default.

Reply:
👍